README.md

```diff
@@ -43,6 +43,37 @@ The services provided are:
 * `api` - convenience environment scoped to the `api` gem in the `/app/api` directory.
 * `sdk` - convenience environment scoped to the `sdk` gem in the `/app/sdk` directory.
 
+## Quick Start
+
+```ruby
+require 'opentelemetry/sdk'
+
+# Set preferred tracer implementation:
+SDK = OpenTelemetry::SDK
+
+factory = OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
+factory.add_span_processor(
+  SDK::Trace::Export::SimpleSpanProcessor.new(
+    # TODO: opentelemetry-python uses ConsoleSpanExporter:
+    SDK::Trace::Export::InMemorySpanExporter.new
+  )
+)
+
+tracer = factory.tracer('trace_name', _version = 'semver:1.0')
```

Comment:
`version` is a positional argument on `TracerFactory#tracer`. I'm uncertain what the tracer name should be for user _application code_, but we should probably use that here. My guess is that it should be the application name, but I'd be open to any other opinions. If we go with that, maybe `trace_name` could be some fictitious application name, such as `my_app`, `my_webstore`, or something slightly more imaginative.

Reply:
From the [OTEP](https://github.com/open-telemetry/oteps/blob/master/text/0016-named-tracers.md):
> If a library (or application) has instrumentation built-in, it is both the instrumenting and instrumented library and should pass its own name here

So yeah, application name appears to be correct.

Reply:
I've updated to use the app or gem name and version (from `Gem.loaded_specs[]`) when creating a tracer.